src/plugins/intel_cpu/tests/functional/behavior/export_import.cpp

```diff
@@ -62,4 +62,47 @@ TEST(ExportImportTest, ExportOptimalNumStreams) {
         EXPECT_EQ(nstreams_latency_original, nstreams_latency_imported);
     }
 }
+
+class ExportOptimalNumStreamsTest : public ::testing::TestWithParam<std::string> {};
+
+TEST_P(ExportOptimalNumStreamsTest, OptimalNumStreamsThroughput) {
+    auto original_model = MakeMatMulModel();
+    std::string deviceName = GetParam();
+    ov::Core core;
+    auto tput_mode = ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT);
+
+    auto original_tp_network = core.compile_model(original_model, deviceName, tput_mode);
+    auto nstreams_tp_original = original_tp_network.get_property(ov::num_streams.name()).as<std::string>();
+
+    std::stringstream exported_stream;
+    original_tp_network.export_model(exported_stream);
+    {
+        std::stringstream ss(exported_stream.str());
+        auto imported_tp_network = core.import_model(ss, deviceName, tput_mode);
+        auto nstreams_tp_imported = imported_tp_network.get_property(ov::num_streams.name()).as<std::string>();
+        EXPECT_EQ(nstreams_tp_original, nstreams_tp_imported);
+    }
+}
+
+TEST_P(ExportOptimalNumStreamsTest, OptimalNumStreamsLatency) {
```

Comment:
If this test instance completely replicates the TEST on the line 36, please remove the TEST on the line 36.
If not, please add "smoke_" prefix for the test on the line 36.

Reply:
@dmitry-gorokhov
I am not sure how much we are in hurry to merge this fix, but this can be considered as a minor one.

Reply:
Ok, removed replicated test.

Reply:
@EgorDuplensky This is "must have" PR for 2023.0 release to fix SPR performance issue and sync inference issue.